dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -40,114 +44,127 @@ TaskFunction createComplexLayoutScrollPerfTest({
+TaskFunction createUiKitViewScrollPerfNonIntersectingTest({bool? enableImpeller, bool unittestMode = false}) {
   return PerfTest(
     '${flutterDirectory.path}/dev/benchmarks/platform_views_layout',
     'test_driver/uikit_view_scroll_perf_non_intersecting.dart',
     'platform_views_scroll_perf_non_intersecting',
     testDriver: 'test_driver/scroll_perf_non_intersecting_test.dart',
     needsFullTimeline: false,
     enableImpeller: enableImpeller,
+    unittestMode: unittestMode,
   ).run;
 }
 
-TaskFunction createAndroidTextureScrollPerfTest({bool? enableImpeller}) {
+TaskFunction createAndroidTextureScrollPerfTest({bool? enableImpeller, bool unittestMode = false}) {
   return PerfTest(
     '${flutterDirectory.path}/dev/benchmarks/platform_views_layout',
     'test_driver/android_view_scroll_perf.dart',
     'platform_views_scroll_perf',
     testDriver: 'test_driver/scroll_perf_test.dart',
     needsFullTimeline: false,
     enableImpeller: enableImpeller,
+    unittestMode: unittestMode,
   ).run;
 }
 
-TaskFunction createAndroidViewScrollPerfTest() {
+TaskFunction createAndroidViewScrollPerfTest({bool unittestMode = false}) {
   return PerfTest(
     '${flutterDirectory.path}/dev/benchmarks/platform_views_layout_hybrid_composition',
     'test_driver/android_view_scroll_perf.dart',
     'platform_views_scroll_perf_hybrid_composition',
     testDriver: 'test_driver/scroll_perf_test.dart',
+    unittestMode: unittestMode,
   ).run;
 }
 
-TaskFunction createHomeScrollPerfTest() {
+TaskFunction createHomeScrollPerfTest({bool unittestMode = false}) {
   return PerfTest(
     '${flutterDirectory.path}/dev/integration_tests/flutter_gallery',
     'test_driver/scroll_perf.dart',
     'home_scroll_perf',
+    unittestMode: unittestMode,
   ).run;
 }
 
-TaskFunction createCullOpacityPerfTest() {
+TaskFunction createCullOpacityPerfTest({bool unittestMode = false}) {
   return PerfTest(
     '${flutterDirectory.path}/dev/benchmarks/macrobenchmarks',
     'test_driver/run_app.dart',
     'cull_opacity_perf',
     testDriver: 'test_driver/cull_opacity_perf_test.dart',
+    unittestMode: unittestMode,
   ).run;
 }
 
-TaskFunction createCullOpacityPerfE2ETest() {
+TaskFunction createCullOpacityPerfE2ETest({bool unittestMode = false}) {
   return PerfTest.e2e(
     '${flutterDirectory.path}/dev/benchmarks/macrobenchmarks',
     'test/cull_opacity_perf_e2e.dart',
+    unittestMode: unittestMode,
   ).run;
 }
 
-TaskFunction createCubicBezierPerfTest() {
+TaskFunction createCubicBezierPerfTest({bool unittestMode = false}) {
   return PerfTest(
     '${flutterDirectory.path}/dev/benchmarks/macrobenchmarks',
     'test_driver/run_app.dart',
     'cubic_bezier_perf',
     testDriver: 'test_driver/cubic_bezier_perf_test.dart',
+    unittestMode: unittestMode,
   ).run;
 }
 
-TaskFunction createCubicBezierPerfE2ETest() {
+TaskFunction createCubicBezierPerfE2ETest({bool unittestMode = false}) {
```

Comment:
Why do we need to add this parameter to all these methods that are never called with `unittestMode = true` and always rely on the default?

Reply:
It was to make the parameter available consistently across all the test methods. The parameter was removed from everywhere.